test/resolver.content.js

```diff
@@ -165,7 +165,8 @@ test('resolver.content() - throwable:true - remote responds with http 500 - shou
     try {
         await content.resolve(outgoing);
     } catch (error) {
-        t.equal(error.output.statusCode, 500);
+        t.equal(error.statusCode, 500);
```

Comment:
this error is our new decorated status code on the root of the error object, this one always reflects the correct status code including 3xx codes